lib/rules/require-explicit-emits.js

```diff
@@ -71,6 +71,25 @@ function getNameParamNode(node) {
   return null
 }
 
+/**
+ * Check if the given name matches defineEmitsNode variable name
+ * @param {string} name
+ * @param {CallExpression | undefined} defineEmitsNode
+ * @returns {boolean}
+ */
+function verifyEmitVariableName(name, defineEmitsNode) {
```

Comment:
Please rename this function to `isEmitVariableName`, see https://github.com/vuejs/eslint-plugin-vue/pull/2340#discussion_r1425643428.